src/util/nestedArray/nestedArray.test.ts

```diff
@@ -0,0 +1,15 @@
+import {nestedIndexOf} from "./nestedArray";
+
+test.each`
```

Comment:
Please improve the formatting here as well, so `test.each` actually increases readability.